cyclonedx-lib/build.xml

```diff
@@ -107,12 +107,35 @@
         <target name="jar">
                 <mkdir dir="build/jar"/>
                 <jar destfile="build/jar/temurin-gen-sbom.jar" basedir="build/classes">
-                        <manifest>
-                                <attribute name="Main-Class" value="temurin.sbom.TemurinGenSBOM"/>
-                        </manifest>
+                <manifest>
+                <attribute name="Main-Class" value="temurin.sbom.TemurinGenSBOM"/>
+                </manifest>
                 </jar>
         </target>
 
+        <target name="buildSignSBOM">
+                <mkdir dir="build/classes"/>
+                <javac srcdir="src" destdir="build/classes" classpath="build/jar/cyclonedx-core-java.jar" includeantruntime="false"/>
```

Comment:
the TemurinSignSBOM.java needs to be in a different src and output build/classes folder, as we need to build it into a separate jar. So say:
sign_src
and build/sign_classes